src/test/java/org/glassfish/logging/annotation/LogMessagesResourceBundleGeneratorTest.java

```diff
@@ -31,21 +31,24 @@
 import javax.tools.ToolProvider;
 import javax.tools.JavaCompiler.CompilationTask;
 
-import junit.framework.TestCase;
-
 import org.glassfish.annotation.processing.logging.LogMessagesResourceBundleGenerator;
 import org.glassfish.annotation.processing.logging.LoggerInfoMetadataGenerator;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import com.foo.bar.Chocolate;
 import com.foo.bar.JavaBean;
 
-public class LogMessagesResourceBundleGeneratorTest extends TestCase {
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class LogMessagesResourceBundleGeneratorTest {
 
     private static final String  BASE_PATH = "./src/test/java/com/foo/bar";
-    
+
+    @BeforeEach
     protected void setUp() {
-        System.out.println("Starting test " + this.getName());
+        System.out.println("Starting test " + this.getClass().getName());
```

Comment:
Test name is usually a method name (it can be even changed in Junit5). You can add an argument to the method, see https://junit.org/junit5/docs/5.0.0/api/org/junit/jupiter/api/TestInfo.html

Reply:
Updated to use `TestInfo`.